apps/src/lab2/views/Lab2.tsx

```diff
@@ -12,18 +12,21 @@ import ProjectContainer from '../projects/ProjectContainer';
 import MetricsAdapter from './MetricsAdapter';
 import LabViewsRenderer from './LabViewsRenderer';
 import DialogManager from './dialogs/DialogManager';
+import ThemeWrapper from './ThemeWrapper';
 
 const Lab2: React.FunctionComponent = () => {
   return (
     <Provider store={getStore()}>
-      <Lab2Wrapper>
-        <DialogManager>
-          <MetricsAdapter />
-          <ProjectContainer channelId={getStandaloneProjectId()}>
-            <LabViewsRenderer />
-          </ProjectContainer>
-        </DialogManager>
-      </Lab2Wrapper>
+      <ThemeWrapper>
```

Comment:
Are we worried about too many wrappers at some point? :)

Reply:
Hahah :) yeah, I think if we can keep isolating all the hierarchical wrapping to just this component, then we should be okay since it's all managed in one place